### PR TITLE
Clarify difference between location.href & location.replace()

### LINKED ISSUE #5523: Issue with "Location: href": Difference between navigation and redirection?
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Location/href

#### What information was incorrect, unhelpful, or incomplete?
"Setting the value of href navigates to the provided URL. If you want redirection, use (some other thing)." - It would be helpful to explain what the difference between navigation and redirection is. Google results are not immediately helpful because Google is not very good for such queries.

#### Specific section or headline?
the top

#### What did you expect to see?
That the difference is explained, or at least links to what navigation and redirection are.

#### Did you test this? If so, how?
n/a

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/location/href`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Location/href
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/location/href/index.html
* Last commit: https://github.com/mdn/content/commit/56d9a678477f0e79232caa8169114519bf02a44a
* Document last modified: 2021-02-19T19:41:18.000Z

</details>

### PR DESCRIPTION
**Pull request by rbrishabh**
Fixes #5523 
Hi  @sideshowbarker, I think just adding the link for redirection should give users enough context to users on what it is, and why they might want to use it!

Do you think this fix works? 
let me know if any changes are required!
Thank you!

### CODE REVIEW

**Review by sideshowbarker — CHANGES_REQUESTED** [review]
This change does not seems sufficient for resolving #5523. The issue description for that issue asked that the difference between redirection and navigation in this context be explained.

I recommend reading https://developer.mozilla.org/en-US/docs/Web/API/Location/replace and https://html.spec.whatwg.org/multipage/history.html#dom-location-replace-dev and then updating this patch by adding an explanation based on what those other documents explain.

**Comment by sideshowbarker** [comment]
> I think just adding the link for redirection should give users enough context to users on what it is, and why they might want to use it!

I don’t think can agree with that — I don’t most users would actually find that just having that link — and reading the content of https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections — would given them enough context.

**Comment by rbrishabh** [comment]
Sure @sideshowbarker! I'm on it!
Thanks for the quick review

**Comment by rbrishabh** [comment]
Hey @sideshowbarker ! Does this look better now?
if you want me to provide more context here, just let me know, happy to make the changes!

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/api/location/href/index.html [hunk 1]
```diff
@@ -15,7 +15,11 @@
   URL, and allows the href to be updated.</p>
 
 <p>Setting the value of <code>href</code> <em>navigates</em> to the provided URL. If you
-  want <em>redirection</em>, use {{domxref("Location/replace","Location.replace()")}}.</p>
+  want <a href="/en-US/docs/Web/HTTP/Redirections"><em>redirection</em></a>, use {{domxref("Location/replace","Location.replace()")}}.
+  The difference from setting the <code>href</code> property value is that when using the
+  <code>Location.replace()</code> method, after navigating to the given URL the current page will not be saved
+  in session <a href="/en-US/docs/Web/API/History"><em>History</em></a>, meaning the user won't be
+  able to use the back button to navigate to it.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/href/index.html:22]:
```suggestion
  want <em>redirection</em>, use {{domxref("Location/replace","location.replace()")}}. The difference from setting the <code>href</code> property value is that when using the <code>location.replace()</code> method, after navigating to the given URL, the current page will not be saved in session <a href="/en-US/docs/Web/API/History_API">history</a> — meaning the user won't be able to use the back button to navigate to it.</p>
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Rishabh, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

**Pull request merged**